src/cli/ionic-config.ts

```diff
@@ -26,16 +26,20 @@ export async function readConfig(): Promise<TelemetryConfig> {
   return config;
 }
 
-export async function writeConfig(config: TelemetryConfig): Promise<void> {
+export async function writeConfig(config: TelemetryConfig): Promise<boolean> {
+  let result = false;
   try {
     await getCompilerSystem().createDir(defaultConfigDirectory(), { recursive: true });
     await getCompilerSystem().writeFile(defaultConfig(), JSON.stringify(config));
```

Comment:
The [Ionic CLI](https://github.com/ionic-team/ionic-cli/blob/develop/packages/@ionic/cli-framework/src/lib/config.ts#L97) indents the config file with two spaces. We should match that for consistency:

```suggestion
    await getCompilerSystem().writeFile(defaultConfig(), JSON.stringify(config, null, 2));
```